pages/touring-series/[slug].vue

```diff
@@ -66,6 +66,59 @@ watch(data, (newVal, oldVal) => {
   series.value = _get(newVal, 'otherSeriesUpcoming', {})
 })
 
+// Get data for Image or Carousel at top of page
+const parsedImage = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!page.value.imageCarousel) {
+    return []
+  }
+  return page.value.imageCarousel
+})
+
+// Transform data for Carousel
+const parsedCarouselData = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!parsedImage.value) {
+    return []
+  }
+  // map image to item, map creditText to credit
+  return parsedImage.value.map((rawItem, index) => {
+    return {
+      // touring-series-ftva/
+      item: [{ ...rawItem.image[0], kind: 'image' }], // Carousels on this page are always images, no videos
+      credit: rawItem?.creditText,
+    }
+  })
+})
+
+// Transform data for Other Series Section
+// This section only shows 3 items max, and prioritizes upcoming events over ongoing
+const parsedOtherSeries = computed(() => {
+  // fail gracefully if data does not exist (server-side)
+  if (!series.value)
+    return []
+
+  let otherSeries = series.value
+  // Remove current series from list
+  otherSeries = otherSeries.filter(item => !item.uri.includes(route.params.slug))
+  // Get first 3 events
+  otherSeries = otherSeries.slice(0, 3)
+
+  // Transform data
+  otherSeries = otherSeries.map((item, index) => {
+    return {
+      ...item,
+      to: `/${item.uri}`, // remove 'series/' from uri
```

Comment:
remove the comment

Reply:
<img width="690" height="106" alt="Screenshot 2025-10-21 at 4 14 21 PM" src="https://github.com/user-attachments/assets/2954c519-7325-4944-a0fa-3c0545bbdecb" />